tests/playwright-tests/general-public/pages/HomePage.ts

```diff
@@ -4,16 +4,26 @@ import environment from '../../utils/env';
 export default class HomePage {
   readonly page: Page;
   readonly glossary: Locator;
+  readonly exploreFindStatistics: Locator;
```

Comment:
I would keep the names of these locators as consistent as we can.  I see that where you've got link locators elsewhere, you call them "SomethingSomethingLink", so I'd be tempted to call this one "exploreFindStatisticsLink" in the same pattern.